encoder/vaapiencpicture.h

```diff
@@ -124,6 +124,8 @@ createMiscObject(VAEncMiscParameterType miscType, T * &bufPtr)
     if (obj) {
         misc->type = miscType;
         bufPtr = (T *) (misc->data);
+        if (bufPtr)
```

Comment:
The data's prototype is array.  misc->data will get the array address. if misc is not null, misc->data always be a valid value.